sysctl.d/50-default.conf

```diff
@@ -30,6 +30,9 @@ net.ipv4.conf.all.accept_source_route = 0
 # Promote secondary addresses when the primary address is removed
 net.ipv4.conf.all.promote_secondaries = 1
 
+# ping(8) without CAP_NET_ADMIN and CAP_NET_RAW
+net.ipv4.ping_group_range = 0 2147483647
```

Comment:
please add a brief comment here why we use 2^31-1 here, and not 2^32-1, i.e. that the kernel doesn't allow it, and we reserve the range > 2^31 range anyway.

Reply:
Done. I also mentioned it in the commit message.